src/components/buttons/rainbow-button/RainbowButton.js

```diff
@@ -102,12 +102,14 @@ const RainbowButton = ({
           type={type}
           width={width}
         />
-        <ButtonContent type={type}>
-          {type === RainbowButtonTypes.addCash && <AddCashIcon />}
-          <ButtonLabel type={type}>
-            {type === RainbowButtonTypes.addCash ? 'Add Cash' : label}
-          </ButtonLabel>
-        </ButtonContent>
+        <ButtonPressAnimation disabled={disabled || ios} onPress={onPress}>
```

Comment:
I don't know how I feel about making this button be a BPA inside a BPA on iOS for no reason.
Should we have a .android.js version of this component instead?